sdk/springappdiscovery/Azure.ResourceManager.SpringAppDiscovery/api/Azure.ResourceManager.SpringAppDiscovery.netstandard2.0.cs

```diff
@@ -0,0 +1,414 @@
+    {
+        protected ErrorSummaryCollection() { }
+        public virtual Azure.Response<bool> Exists(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> Get(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource>> GetAsync(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.NullableResponse<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> GetIfExists(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.NullableResponse<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource>> GetIfExistsAsync(string errorSummaryName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.SpringAppDiscovery.ErrorSummaryResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class ErrorSummaryData : Azure.ResourceManager.Models.ResourceData
+    {
+        public ErrorSummaryData() { }
+        public Azure.ResourceManager.SpringAppDiscovery.Models.ErrorSummariesProperties Properties { get { throw null; } set { } }
```

Comment:
If we choose to change the name of `ErrorSummaryData`, we need to apply similar change to this type (not this property name, but its type name)